float64 x
float64 y
float64 angular_vel             # Anagular velocity [rad/s]
float64 linear_vel             # Linear velocity [rad/s]
float64 radius                  # Radius with added margin [m]
float64 theta                   # Angle
int32 id